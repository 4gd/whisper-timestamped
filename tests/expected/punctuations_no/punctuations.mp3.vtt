WEBVTT

00:00.380 --> 00:02.400
Dis-moi, est-ce que l'avion vole?

