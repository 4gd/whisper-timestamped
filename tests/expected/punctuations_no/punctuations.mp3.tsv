start	end	text
380	2400	Dis-moi, est-ce que l'avion vole?
